plugins/source/aws/docs/tables/aws_iam_group_services_last_accessed.md

```diff
@@ -0,0 +1,23 @@
+# Table: aws_iam_group_services_last_accessed
+
+https://docs.aws.amazon.com/IAM/latest/APIReference/API_ServiceLastAccessed.html
+
+The primary key for this table is **_cq_id**.
+
+
+## Columns
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_source_name|String|
+|_cq_sync_time|Timestamp|
+|_cq_id (PK)|UUID|
+|_cq_parent_id|UUID|
+|account_id|String|
+|service_name|String|
```

Comment:
Would account_id + service_name be a possible primary key here, or could there be cases where that is not unique? Is there any combination that will definitely be uniquely identifying?